service/src/event/store.rs

```diff
@@ -20,22 +20,12 @@ impl recon::Store for CeramicEventService {
     async fn insert_many(
         &self,
         items: &[ReconItem<Self::Key>],
-    ) -> ReconResult<recon::InsertResult> {
+    ) -> ReconResult<recon::InsertBatch<EventId>> {
         let res = self
             .insert_events(items, DeliverableRequirement::Asap)
             .await?;
-        let mut keys = vec![false; items.len()];
-        // we need to put things back in the right order that the recon trait expects, even though we don't really care about the result
```

Comment:
why don't we need to do this anymore?

Reply:
we always just ignored it in the past and it's a bit annoying to recompute since everything can get reordered before being inserted